docs/root/configuration/http_conn_man/headers.rst

```diff
@@ -118,15 +118,22 @@ should be replaced by backslash-double-quote (\").
 
 The following keys are supported:
 
-1. ``By`` The Subject Alternative Name (SAN) of the current proxy's certificate.
+1. ``By`` The Subject Alternative Name (URI type SAN) of the current proxy's certificate.
 2. ``Hash`` The SHA 256 diguest of the current client certificate.
-3. ``SAN`` The SAN field (URI type) of the current client certificate.
+3. ``SAN`` The URI type SAN field of the current client certificate.
```

Comment:
Since we'll have 2 types of Subject Alternative Name, perhaps we should change `SAN` to `URI` (or more likely, support both and document `SAN` as deprecated)? cc @mattklein123

Reply:
Sure SGTM